tests/flow/test_crash_handler.py

```diff
@@ -0,0 +1,19 @@
+from common import *
+from index_utils import *
+from constraint_utils import *
+
+GRAPH_ID = "constraints"
+
+class testCrashHandler():
+    def __init__(self):
+        self.env, self.db = Env(enableDebugCommand=True)
+
+    def test_crash_handler(self):
+        try:
+            self.db.execute_command("DEBUG", "SEGFAULT")
+        except:
+            pass
+        logfilename = self.env.envRunner._getFileName("master", ".log")
```

Comment:
Should we be waiting for the `redis-server` process to exit before looking into the log file ?
I think it would be better to see that indeed the process no longer exists before accessing the file.